# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/btrfs-progs/hotfix.patch.musl
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- btrfs-progs-v5.14/common/utils.c.vanilla	2021-10-10 18:34:30.266597977 +0200
+++ btrfs-progs-v5.14/common/utils.c	2021-10-10 18:34:37.536597544 +0200
@@ -17,6 +17,8 @@
  * Boston, MA 021110-1307, USA.
  */
 
+#define _LINUX_SYSINFO_H
+
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
